### PR TITLE
Add community tools page and update networks links

### PR DESCRIPTION
**Pull request by smb2796**

### CODE REVIEW

File: docs/community/community-built-tools.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+---
+title: Community Built Tools & Resources 
+sidebar_label: Tools & Resources 
+---
+
+Various UMA community members and SuperUMANs have created different tools and resources as public consumption. To have your contribution added to this list, please reach out in Discord.
```
Comment (ChandlerDeKock) [inline docs/community/community-built-tools.md:6]:
```suggestion
Various UMA community members and SuperUMANs have created different tools and resources for public consumption. To have your contribution added to this list, please reach out in Discord.
```

File: docs/developers/deploy-on-polygon.md [hunk 2]
```diff
@@ -138,6 +138,6 @@ Once your contract is deployed, you can head over the specific Financial Product
 
 In the case of this launch demo, we used the `CoveredCallLongShortPairFinancialProductLibrary` to link the new LSP contract with a strike price. 
 
-In the `longShortPair address` field, you can put in your newly created LSP contract (output from the deployment script) and in the `strikePrice` you can put the strike price that the financial product library will use to calculate the valeu fo the long and short tokens (note: the stoke price should be converted to an 18 decimal number)
+In the `longShortPair address` field, you can put in your newly created LSP contract (output from the deployment script) and in the `strikePrice` you can put the strike price that the financial product library will use to calculate the value for the long and short tokens (note: the strike price should be converted to an 18 decimal number)
```
Comment (ChandlerDeKock) [inline docs/developers/deploy-on-polygon.md:141]: Good spot

**Review by ChandlerDeKock — APPROVED** [review]
2 small suggestions. Otherwise looks good

File: docs/community/community-built-tools.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+Please note that these tools are community built and maintained and Risk Labs does not assume responsiblity for their security or functionality.
```
[the remark this answers is not in the document]
Comment (smb2796) [reply docs/community/community-built-tools.md:8]: Good catch!

**Pull request merged**